app/assets/javascripts/discourse/tests/acceptance/composer-draft-saving-test.js

```diff
@@ -0,0 +1,53 @@
+import {
+  acceptance,
+  exists,
+  query,
+} from "discourse/tests/helpers/qunit-helpers";
+import { click, fillIn, visit } from "@ember/test-helpers";
+import { test } from "qunit";
+import I18n from "I18n";
+
+acceptance("Composer - Draft saving", function (needs) {
+  needs.user();
+
+  let callNumber = 0;
+  needs.pretender((server, helper) => {
```

Comment:
Are we 100% sure this won't leak to other tests? Otherwise this will introduce a lot of heisentests.

Reply:
Hmm, I'm pretty sure this won't leak but not 100% sure. I built it several times here, and I don't see problems by now. I'll be monitoring the build for some time after merging.